discovery-provider/src/tasks/index_aggregate_track.py

```diff
@@ -0,0 +1,181 @@
+import logging
+
+from src.tasks.celery_app import aggregate_worker, celery, celery_worker
+from src.tasks.index_aggregate_user import get_latest_blocknumber
+
+logger = logging.getLogger(__name__)
+
+AGGREGATE_TRACK = "aggregate_track"
+
+# UPDATE_AGGREGATE_TRACK_QUERY
+# Get a lower/higher bound blocknumber to check for new save/repost activity
```

Comment:
I see the upper bound but not the lower bound. I think in this case it would not necessarily look for tracks that have recently changed, it would look at all tracks. I think we'd want something more similar to aggregate_user.

Reply:
Good point!  This was left over from the previous commit. Updating now!